src/inmanta/util/__init__.py

```diff
@@ -476,7 +477,7 @@ def _custom_json_encoder(o: object) -> Union[ReturnTypes, "JSONSerializable"]:
     if isinstance(o, JSONSerializable):
         return o.json_serialization_step()
 
-    if isinstance(o, uuid.UUID):
+    if isinstance(o, (uuid.UUID, Url)):
```

Comment:
Why do we need this?

Reply:
Good question actually, @bartv do you know? I think it can't hurt either way, but I'm not sure what changed to require it now.

Reply:
I think because before these Urls were strings and not they are their own class.